rmgpy/molecule/group.py

```diff
@@ -1387,7 +1406,10 @@ def get_extensions(self, r=None, basename='', atm_ind=None, atm_ind2=None, n_spl
                        extents=list, RnH=list, typ=list)
 
         extents = []
-
+        if r_bonds is None:
+            r_bonds=[1,2,3,1.5,4]
```

Comment:
Can you run a formatter through the changes

Reply:
I've made the changes I think you're looking at, at least.